protocol/src/main/java/org/jboss/as/protocol/ProtocolConnectionConfiguration.java

```diff
@@ -103,12 +102,13 @@ public void setEndpoint(Endpoint endpoint) {
         this.endpoint = endpoint;
     }
 
+    @Deprecated
     public OptionMap getOptionMap() {
-        return optionMap;
+        return OptionMap.EMPTY;
     }
 
+    @Deprecated
     public void setOptionMap(OptionMap optionMap) {
-        this.optionMap = optionMap;
     }
```

Comment:
@darranl this change to how optionMap is handled looks to me to be a breaking change. RemoteDomainConnectionService.start is using this to configure heartbeats and a read timeout.